oaknational/Oak-Web-Application
# Feat/lesq 914/share all to pupils [LESQ-914]

## Description

Music year: {owa_music_year}

- remove hardcoded list of subjects which can be shared to the new pupil experience
- remove conditional usage of new pupil experience for share links
- remove code related to selecting resources to share as the new pupil experience does not support this

## How to test

1. Go to https://deploy-preview-2553--oak-web-application.netlify.thenational.academy/teachers/programmes/rshe-pshe-secondary-ks4-l/units/being-safe-informed-consent-6fd6/lessons/sexual-consent-and-capacity-to-consent-61j3ec/share?preselected=all
2. Click on preview as a pupil, or one of the share options
3. You should be directed to the new pupil experience version of the lesson instead of the home page

## Checklist

- [ ] All legacy content can be shared to the pupil view
- [ ] Remove all the custom code that was handling subject by subject restrictions on share
- [ ] Change header to be “Share your lesson” and remove “Select online activities”
